template for new rust files

colin-kiegel opened the pull request:
This is the template I used for new files so far. What do you think?

colin-kiegel committed:
template for new rust files

Nercury commented on .template.rs:
```diff
@@ -0,0 +1,18 @@
+///
+/// @author NAME <EMAIL>
+
+/////////////
```
Can we do away without these? It's the first time I am seeing such comments in Rust code.

colin-kiegel committed:
rust-template: simplified header

colin-kiegel commented:
Ok, simplified it. :-)
- I removed import/export, as you suggested
- I also removed authors from the header - since we will probably distribute a global AUTHORS file and will also be visible via git-blame
